Makefile

```diff
@@ -0,0 +1,34 @@
+PATH_TO_INVENTORY=../integreatly_dev/inventories/hosts
+PATH_TO_VAULT_FILE=~/.vp
+OPENSHIFT_MASTER=integreatly-cicd.skunkhenry.com
+OPENSHIFT_USERNAME=redacted
+OPENSHIFT_PASSWORD=redacted
+GIT_REF=$(shell sh -c "git rev-parse --short HEAD")
+TOWER_OPENSHIFT_PROJECT=tower-e2e-${GIT_REF}
```

Comment:
Using git ref here to generate a namespace. It should be different each time. If there's a case where it's not and the tests have failed previously we could see idempotency failures.

Reply:
The more I think about this the more I don't want to use GIT_REF.

Reply:
Setting a default looks good, doesn't really need to be made unique if you don't want to use the GITREF. We should follow the pattern we have now for other e2e tests of overriding the e2e target specifically for prow. This will allow us to set the TOWER_OPENSHIFT_PROJECT env var to match the prow namespace see https://github.com/integr8ly/heimdall/blob/master/Makefile#L48-L59

```
.PHONY: test/e2e/prow
test/e2e/prow: export TOWER_OPENSHIFT_PROJECT=OPENSHIFT_BUILD_NAMESPACE
test/e2e/prow: test/e2e

.PHONY: test/e2e
test/e2e: 
   @./test/ci/e2e.sh
```